apps/web/src/components/Navigationbar.tsx

```diff
@@ -0,0 +1,82 @@
+"use client"
+
+import React from 'react'
+import {  Navbar,   NavbarBrand,   NavbarContent,   NavbarItem,   NavbarMenuToggle,  NavbarMenu,  NavbarMenuItem} from "@nextui-org/navbar";
+import {Button} from "@nextui-org/button";
+import Link  from 'next/link';
+
+
+function Navigationbar() {
+    const [isMenuOpen, setIsMenuOpen] = React.useState(false);
+
+    const menuItems = [
+        "Homepage",
+        "About Us",
+        "Contact Us"
+    ];
+  return (
+        <div className=''>
+            <Navbar onMenuOpenChange={setIsMenuOpen} isBordered>
+                <NavbarContent justify='start'>
+                    <NavbarMenuToggle
+                    aria-label={isMenuOpen ? "Close menu" : "Open menu"}
+                    className="sm:hidden"
+                    />
+                    <NavbarBrand>
+                    <p className="font-bold text-inherit">NoterAi</p>
+                    </NavbarBrand>
+                </NavbarContent>
+
+                <NavbarContent className="hidden sm:flex gap-4" justify="center">
+                    <NavbarItem>
+                    <Link color="foreground" href="/">
+                        Homepage
+                    </Link>
+                    </NavbarItem>
+
+                    <NavbarItem isActive>
```

Comment:
keep this filled as a state variable, or remove altogether

isActivate var one...